generators/embeddable-graphic/templates/src/js/scripts.js

```diff
@@ -0,0 +1,25 @@
+import $ from 'jquery';
+import pym from 'pym.js';
+
+
+const pymChild = new pym.Child();
+
+$(() => {
```

Comment:
Do we need to wrap this func in jquery or is that just so the linter doesn't yell?